sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -1051,6 +1053,22 @@ class AstBuilder extends SqlBaseBaseVisitor[AnyRef] with Logging {
   }
 
   /**
+   * Create a name LTRIM for TRIM(Leading), RTRIM for TRIM(Trailing), TRIM for TRIM(BOTH)
+   */
+  private def getTrimFuncName(ctx: FunctionCallContext, optType: Int): String = {
+    if (ctx.qualifiedName.getText.toLowerCase != "trim") {
+      throw new ParseException(s"The specified function ${ctx.qualifiedName.getText}  " +
```

Comment:
Hi, @kevinyu98 .
There are two spaces after `${ctx.qualifiedName.getText}`. We had better to reduce that into one space.